lib/chain/src/ChainStore.js

```diff
@@ -1667,6 +1671,114 @@ class ChainStore {
         });
         return didImpact;
     }
+
+    getLiquidityPoolByID(pool_id) {
```

Comment:
this is unnecessary, it does not do anything extra compared to getObject. This might be misleading (users expecting something different than other methods). I suggest to remove this

Reply:
right.

Reply:
Are you talking about "getLiquidityPoolByID" only or all the functions?

Reply:
Just this one. Can use getObject directly

Reply:
Just FWIW the `get_liquidity_pools` API has features which are not included in `get_objects`.

Reply:
Thanks @abitmore; I did not compare get_objects with get_liquidity_pools so far but will be checking them, these functions in this PR were made to coverup a scope that's required for the web UI, will get developed by time.